src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -106,12 +108,59 @@ public function indexAction(Request $request, OrderFilters $filters)
         );
     }
 
-    //@todo: wip
+    /**
+     * Places an order from BO
+     *
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function placeAction(Request $request)
```

Comment:
Mmm, not sure I understand the verb `place` in this context 🤔 It's actually a creation no? Although I understand the necessity to have a rendering create action and one to manage the saving I'm not sure about this naming

Reply:
Yea, its open for discussion. I thought it would be more confusing if i call it createAction.

Reply:
Since you split the routing I think it will be more understandable to have an explicit name like `processCreateAction` `submitCreateAction` `handleCreateAction` I wonder if we have a convention for those use cases
@matks how is the split between GET/POST of a form managed usually?

Reply:
Verb is fine I think 👍 "to place an order" is OK in english

Reply:
GET and POST form actions are usually the same action, with a `if ($request->isPOST()` in it